schemas/com.snowplowanalytics.snowplow.storage/redshift_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Snowplow Redshift storage configuration",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.storage",
+        "name": "redshift_config",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        },
+        "host": {
+            "type": "string"
+        },
+        "database": {
+            "type": "string"
+        },
+        "port": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 65535
+        },
+        "sslMode": {
+            "type": "string",
+            "enum": ["disable", "require", "verify-ca", "verify-full"]
+        },
+        "schema": {
+            "type": "string"
+        },
+        "username": {
+            "type": "string"
+        },
+        "password": {
+            "type": "string"
+        },
+        "maxError": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 32767
```

Comment:
Max is 10000, per http://docs.aws.amazon.com/redshift/latest/dg/copy-parameters-data-load.html#copy-maxerror